wolox-training/rof-angular
# Implemented change password on profile

## Summary

Implemented change password on profile

## Test Plan

* Log your account or create a new one
* Go to profile on header
* Choose change password
* Change it
* Log again to confirm password has changed.

## Trello Card 

 https://trello.com/c/F8bGzo6i

## Screenshot

![matchpassword](https://cloud.githubusercontent.com/assets/2911971/10680548/2006733c-78f7-11e5-8cc2-c7f9d329f9bf.png)
